File: app/src/main/java/com/shevelev/comics_viewer/activities/main/IChangeModeHandlerView.java
package com.shevelev.comics_viewer.activities.main;

import android.view.View;

public interface IChangeModeHandlerView
{
    void onAllComicsClicked(View allComicsControl, View recentComicsControl);
    void onRecentComicsClicked(View allComicsControl, View recentComicsControl);

    void initState(View allComicsControl, View recentComicsControl);
}

File: app/src/main/java/com/shevelev/comics_viewer/activities/main/IChangeModeHandlerView.kt
package com.shevelev.comics_viewer.activities.main

import android.view.View

interface IChangeModeHandlerView {
    fun onAllComicsClicked(allComicsControl: View, recentComicsControl: View)
    fun onRecentComicsClicked(allComicsControl: View, recentComicsControl: View)
    fun initState(allComicsControl: View, recentComicsControl: View)
}